.travis/package-osx.sh

```diff
@@ -2,6 +2,9 @@
 
 mkdir -p $HOME/artifacts
 
-cd $TRAVIS_BUILD_DIR/BUILD/Install
+cd $TRAVIS_BUILD_DIR/BUILD/Install/SuperCollider
+xattr -cr SuperCollider.app
+codesign --deep --force --verify --verbose --sign "Mac Developer: josh@realizedsound.net" SuperCollider.app
```

Comment:
add `|| exit 1` or something similar after this so the build will fail if codesigning fails